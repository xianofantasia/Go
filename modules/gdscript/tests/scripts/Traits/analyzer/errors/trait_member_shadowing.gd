uses TraitA

var some_abstract_function
