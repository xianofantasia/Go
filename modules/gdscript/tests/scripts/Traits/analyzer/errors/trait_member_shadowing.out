GDTEST_ANALYZER_ERROR
>> ERROR at line 3: "some_abstract_function" variable shadows function in trait "TraitA".
